README.md

```diff
@@ -29,6 +29,12 @@ cargo install cargo-llvm-cov
 cargo llvm-cov
 ```
 
+## Coverage Report
+
+```bash
+cargo llvm-cov --html or cargo llvm-cov --open
```

Comment:
```suggestion
cargo llvm-cov --html # or cargo llvm-cov --open
```
making this a comment would be useful for copy-pasting

Reply:
Good call, updated.